.github/workflows/ci.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - master
```

Comment:
I assume there will be another pass where these references to the master will be removed, right?

Reply:
Yep! I'll likely rebase #430 on top of this after its committed and then remove the old references. This is just to make sure everything continues working in the interim.

Reply:
Cool! Thanks for the confirmation!